src/main/resources/db/changelog/changes/v1.2.0/sql/2022-06-13-01__alter-transaction-item-hold.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE transaction_item_hold
+  ADD COLUMN author VARCHAR(255);
```

Comment:
move the column to transaction_hold.
See how it was done for other fields, for example https://issues.folio.org/browse/MODINREACH-206

Reply:
thanks for hint